package.json

```diff
@@ -2,24 +2,28 @@
   "name": "@c-frame/aframe-particle-system-component",
   "version": "1.1.4",
   "description": "Particle systems for A-Frame.",
-  "main": "index.js",
+  "browser": "index.js",
```

Comment:
browser key is long time deprecated, you should keep main key here, also use relative "./index.js" that is required for some bundler apparently.